dependencies/dependencies.json

```diff
@@ -0,0 +1,26 @@
+[
+    {
+        "library": "boost",
+        "version": "1.81.0"
+    },
+    {
+        "library": "maeparser",
+        "version": "1.3.1"
+    },
+    {
+        "library": "fmt",
+        "version": "6.1.2"
+    },
+    {
+        "library": "qt6",
+        "version": "6.2.5"
```

Comment:
should this be 6.2.7?
https://opengrok.schrodinger.com/xref/mmshare/build_tools/library_definitions.json?r=9c596363#309

Reply:
Ok, so it looks like niether 6.2.7 nor 6.2.6 are available?
```
+ curl -L -o qt6-base-6.2.7.tar.xz https://download.qt.io/archive/qt/6.2/6.2.7/submodules/qtbase-everywhere-opensource-src-6.2.7.tar.xz
...
tar: This does not look like a tar archive
xz: (stdin): File format not recognized
...
```